packages/ember-glimmer/tests/integration/helpers/inline-if-test.js

```diff
@@ -0,0 +1,467 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import { set } from 'ember-metal/property_set';
+import EmberObject from 'ember-runtime/system/object';
+import ObjectProxy from 'ember-runtime/system/object_proxy';
+import { A as emberA } from 'ember-runtime/system/native_array';
+import ArrayProxy from 'ember-runtime/system/array_proxy';
+
+//TEMP: GJ: extract and reuse from elsewhere (these are moving in https://github.com/emberjs/ember.js/pull/12919)
+class AbstractConditionalsTest extends RenderingTest {
+
+  /* abstract */
+  templateFor({ cond, truthy, falsy }) {
+    // e.g. `{{#if ${cond}}}${truthy}{{else}}${falsy}{{/if}}`
+    throw new Error('Not implemented: `templateFor`');
+  }
+
+  renderValues(...values) {
+    let templates = [];
+    let context = {};
+
+    //TODO: GJ: update common `AbstractConditionalsTest` to use this format
+    for (let i = 1; i <= values.length; i++) {
+      templates.push(this.templateFor({ cond: `cond${i}`, truthy: `t${i}`, falsy: `f${i}` }));
```

Comment:
with a slight change, we could reuse this logic for both syntax and inline conditional tests. 

Syntax:
`templates.push(this.templateFor({ cond: `cond${i}`, truthy: `{{t${i}}}`, falsy: `{{f${i}}}` }));`

Inline:
`templates.push(this.templateFor({ cond: `cond${i}`, truthy: `t${i}`, falsy: `f${i}` }));`

Perhaps we should define `AbstractSyntaxConditionalsTest` and `AbstractInlineConditionalsTest`